### PR TITLE
scala 

### PR DESCRIPTION
**Pull request by ThomasDuPlessis**
made some simple changes to allow scala completion (changes were only in eclim-completion.el) which is now supported again in the recent versions of eclim.

### CODE REVIEW

File: eclim.el [hunk 1]
```diff
@@ -508,7 +509,7 @@ the use of eclim to java and ant files."
                (groovy-mode "groovy_src_update")
                (ruby-mode "ruby_src_update")
                (php-mode "php_src_update")
-
+	       ((scala-mode) "scala_src_update")
```
Comment (kleewho) [inline eclim.el]: please fix the indentation here

File: eclim.el [hunk 2]
```diff
@@ -171,7 +171,8 @@ where <encoding> is the corresponding java name for this encoding." e e)))
                                         ("groovy_complete" "Groovy" "Eclipse Groovy Development Tools")
                                         ("ruby_complete" "Ruby" "Eclipse Ruby Development Tools")
                                         ("c_complete" "C/C++" "Eclipse C/C++ Development Tools")
-                                        ("php_complete" "PHP" "Eclipse PHP Development Tools")))))
+                                        ("php_complete" "PHP" "Eclipse PHP Development Tools")
+					("scala_complete" "Scala" "Eclipse Scala Development Tools")))))
```
Comment (kleewho) [inline eclim.el]: please fix indentation here

File: History.txt [hunk 3]
```diff
@@ -20,7 +20,8 @@
  * User commands for adding, listing and removing project natures.
  * Running all JUnit tests in a class or just the test method at point.
  * additional info in modeline about errors and warnings
-
+ *  added support for scala completion
```
Comment (kleewho) [inline History.txt]: All lines here have just one space

File: eclim-completion.el [hunk 4]
```diff
@@ -68,6 +68,8 @@
                (eclim/execute-command "php_complete" "-p" "-f" "-e" "-o"))
               ((javascript-mode js-mode)
                (eclim/execute-command "javascript_complete" "-p" "-f" "-e" "-o"))
+	      (scala-mode
```
Comment (kleewho) [inline eclim-completion.el]: I'm pretty sure that indentation here is also wrong

File: eclim.el (hunk 1 above)
Comment (kleewho) [inline eclim.el]: Is `scala-mode` a function? I think that you should drop one set of `()`

**Comment by kleewho** [comment]
Just minor details from my point of view

**Comment by ThomasDuPlessis** [comment]
hmmm thats odd, seems to be correct in my set up. I'll fix it.

File: eclim.el (hunk 1 above)
Comment (ThomasDuPlessis) [inline eclim.el]: shouldnt be a function, i'll change that.

**Comment by ThomasDuPlessis** [comment]
it should be good now I think. (sorry for the dumb tab mistakes, first time contributing to another persons project on github)

**Comment by kleewho** [comment]
@ThomasDuPlessis and it's one of the first times when I'm reviewing something and I'm not even maintainer so I'd say good job both of us :+1:

**Comment by kleewho** [comment]
So right now it would be nice if you would squash the commits to just one with meaningful description (without something like: fixed tabs) and probably you shouldn't have this merge commit as well.

**Comment by jchochli** [comment]
@ThomasDuPlessis  @kleewho  thanks :)

**Comment by ThomasDuPlessis** [comment]
is this good?

**Comment by kleewho** [comment]
@ThomasDuPlessis Looks good to me, but you have to wait on @jchochli for merge. Thank you :clap:

**Comment by jchochli** [comment]
I'll test what I can and merge by the weekend. Thanks!

**Pull request merged**